password_security/models/res_company.py

```diff
@@ -9,6 +9,7 @@
 class ResCompany(models.Model):
     _inherit = "res.company"
 
+    password_policy_enabled = fields.Boolean(default=False)
```

Comment:
I guess whoever installed this module already, expects it to be on by default, right?
If yes, we need a migration script to set the flag to True on upgrade.
N'est pas?

Reply:
You are right! I pushed a migration script (tested locally on a customer project).